docs/RNCamera.md

```diff
@@ -245,14 +265,22 @@ Value: float from `0` to `1.0`
 
 Specifies the zoom of your camera. The value 0 is no zoom, 1 is maximum zoom. For a medium zoom, for example, you could pass `0.5`.
 
-#### `Android` `permissionDialogTitle`
+#### `Android` `permissionDialogTitle` - Depracated
 
 Starting on android M individual permissions must be granted for certain services, the camera is one of them, you can use this to change the title of the dialog prompt requesting permissions.
 
-#### `Android` `permissionDialogMessage`
+#### `Android` `permissionDialogMessage` - Depracated
```

Comment:
should we just remove it?

Reply:
We could but I think people might want to look at the documentation once they get a warning. I think it's better to remove it once we remove it from the code completely.

Reply:
I also think deprecating in a minor release, Followed by a major release that removed multiple depreciations should be the way to go.